File: Selenium/Project/Activity_3.py
from selenium.webdriver.common.by import By
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("http://alchemy.hguy.co/crm")
    footertext = driver.find_element(By.XPATH,"//a[@id=\"admin_options\"]")
    print("footer_text ", footertext.text)
    driver.close()

File: Selenium/Project/Activity_3.java
package Activity_CRM;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity_3 {
    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
        String footertext = driver.findElement(By.xpath("//a[@id=\"admin_options\"]")).getText();
        System.out.println("footer_text " + footertext);
        driver.close();
    }
}
